README.md

```diff
@@ -1,5 +1,58 @@
-# ssi-dim-wallet-stub
+# SSI DIM Wallet stub application
 
-Stub application for Decentralize Identity Management (DIM) Wallet
+#### Note: *This application can not be used in production. This is designed and developed for local and integration testing only*
```

Comment:
As Tractus-X provides reference implementations only the reference to "production" is a bit misleading.
Maybe just leave it with "This application meant for **local setup** and - integration - **testing** only"

Reply:
Addressed: https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/pull/2/commits/006e091ce7de7a4c265534389e3288e35968c732